### DISCUSSION TITLE
List followers of a user API

### DISCUSSION THREAD

**Discussion by vinijmoura** [community/community#33307] [Apps, API and Webhooks] [answered]
Is it possible API **List followers of a user** returns the follower user date?

https://docs.github.com/en/rest/users/followers#list-followers-of-a-user

**Comment by melvincwng** [comment, marked as answer]
@vinijmoura, based on the [official documentation,](https://docs.github.com/en/rest/users/followers#list-followers-of-a-user) this is the sample response that is returned when we hit the followers endpoint of a particular user:

<pre>
[
  {
    "login": "octocat",
    "id": 1,
    "node_id": "MDQ6VXNlcjE=",
    "avatar_url": "https://github.com/images/error/octocat_happy.gif",
    "gravatar_id": "",
    "url": "https://api.github.com/users/octocat",
    "html_url": "https://github.com/octocat",
    "followers_url": "https://api.github.com/users/octocat/followers",
    "following_url": "https://api.github.com/users/octocat/following{/other_user}",
    "gists_url": "https://api.github.com/users/octocat/gists{/gist_id}",
    "starred_url": "https://api.github.com/users/octocat/starred{/owner}{/repo}",
    "subscriptions_url": "https://api.github.com/users/octocat/subscriptions",
    "organizations_url": "https://api.github.com/users/octocat/orgs",
    "repos_url": "https://api.github.com/users/octocat/repos",
    "events_url": "https://api.github.com/users/octocat/events{/privacy}",
    "received_events_url": "https://api.github.com/users/octocat/received_events",
    "type": "User",
    "site_admin": false
  }
]                     
</pre>

There doesn't seem to be a key/value pair containing the date of following in the JSON response thats returned.

Hence, to answer your question, I don't think it's possible to get the date of following of a particular follower.

Hope this clarifies and answers your question 😊

**Reply by vinijmoura** [reply]
Hello @melvincwng 
I know this API and read the official documentation. My question is: is it possible to return this information?
Example: On Forks(created_at key/value) and Starred(starred_at key/value) APIs I have this information.
Is it possible to change this REST API?

**Reply by melvincwng** [reply]
@vinijmoura, No I don't think that's possible.

Hope this clarifies.

**Reply by vinijmoura** [reply]
How can I open the issue to GitHub Dev Team?
